packages/nextjs/app/builders/0xaa4C60b784E2b3E485035399bF1b1aBDeD66A60f/page.tsx

```diff
@@ -0,0 +1,56 @@
+import { GithubIcon } from "./_components/GithubIcon";
+import { WalletIcon } from "./_components/WalletIcon";
+import { XIcon } from "./_components/XIcon";
+import type { NextPage } from "next";
+import { BlockieAvatar } from "~~/components/scaffold-eth";
+import { getMetadata } from "~~/utils/scaffold-eth/getMetadata";
+
+export const metadata = getMetadata({
+  title: "Danitome24 builder page",
+  description: "Builder page for danitome24 with a little bio and social links.",
+});
+
+const Builder: NextPage = () => {
+  return (
+    <div className="max-w-lg mx-auto mt-10 p-6 bg-white shadow-md rounded-lg">
+      <div className="text-center">
+        <div className="w-24 h-24 rounded-full mx-auto mb-4">
+          <BlockieAvatar address="0xaa4C60b784E2b3E485035399bF1b1aBDeD66A60f" size={120} />
+        </div>
+
+        <h1 className="text-2xl font-bold text-gray-800">danitome24</h1>
+      </div>
+
+      <div className="mt-4 text-center">
```

Comment:
Instead of repeating ```text-gray-700``` in multiple ```<p>``` you could move into the ```<div>``` around the paragraphs.

Reply:
done!